src/include/homestore/replication_service.hpp

```diff
@@ -6,6 +6,7 @@
 
 #include <folly/futures/Future.h>
 
+#include <homestore/homestore.hpp>
```

Comment:
Do we need the header here? We can move it to the cpp file

Reply:
done.